pkg/model/resource/webhooks.go

```diff
@@ -33,6 +33,8 @@ type Webhooks struct {
 
 	// Conversion specifies if a conversion webhook is associated to the resource.
 	Conversion bool `json:"conversion,omitempty"`
+
+	Spoke string `json:"spoke,omitempty"`
```

Comment:
`Spoke` can be multiple, so this should probably be `[]string`. Or do we want the user to execute this command multiple times for each spoke version? One issue would be that we need to ensure that `hub` is not scaffolded again as it may interfere with the already generated code.